plugins/filter_kubernetes/kube_conf.c

```diff
@@ -61,6 +62,15 @@ struct flb_kube *flb_kube_conf_create(struct flb_filter_instance *ins,
         return NULL;
     }
 
+    /* Token Command */
+    cmd = flb_filter_get_property("token_command", ins);
+    if (cmd) {
+        ctx->token_command = cmd;
+    }
+    else {
+        ctx->token_command = NULL;
```

Comment:
I didn't set a default value for this field so we need to get it and then set it by every case. This is because the command may contain blank space so we cannot just map the value from config map.